cli/util.go

```diff
@@ -446,6 +446,16 @@ func newNatsConnUnlocked(servers string, copts ...nats.Option) (*nats.Conn, erro
 		servers = opts.Config.ServerURL()
 	}
 
+	if opts.UserJwt != "" {
```

Comment:
Thank you, to be in line with how things work we should probably add this to the natscontext package also so users can save this in their different contextes.  I can do that later this week if its too much for you to take on

Reply:
It needs to be exposed here also rite?

"github.com/nats-io/jsm.go/natscontext"

Reply:
Yeah that one, so we need a property there and helpers to set it, and helpers to create the nats options once its set.  Then the update here is just adding the flag.

Reply:
I've merged that, if you rebase here and do `GOPRIVATE=github.com/nats-io go get github.com/nats-io/jsm.go@main` it will update your go.mod with that version and tests ehre will pass

Reply:
Think you'll first have to rebase onto cli main then do the go get before it will pass